lib/v8debugapi.js

```diff
@@ -201,6 +201,40 @@ module.exports.create = function(logger_, config_, fileStats_) {
       listeners[num] = listener;
     },
 
+    /**
+     * @param {Breakpoint} breakpoint
+     * @param {Function} callback
+     */
+    log: function(breakpoint, format, shouldStop) {
+      var logsThisSecond = 0;
+      var timesliceEnd = Date.now() + 1000;
+      var num = breakpoints[breakpoint.id].v8Breakpoint.number();
+      var listener = onBreakpointHit.bind(
+          null, breakpoint, function(err) {
```

Comment:
nit: This is kind of a big function; could this be broken out into a separate function-factory entirely to so that is property isn't as busy?